app/models/image.rb

```diff
@@ -662,7 +662,10 @@ def validate_image_name
   # to the :image field.  Returns true if the file is successfully saved.
   def save_to_temp_file
     result = true
-    unless upload_temp_file.present?
+    # disable cop; In theory upload_temp_file could be
+    # a blank non-empty string (e.g. " ").
+    # Don't fix unless...present?; it isn't broken.
+    unless upload_temp_file.present? # rubocop:disable Rails/Blank
```

Comment:
`present?` is literally defined as `!blank?` (see https://github.com/rails/rails/blob/main/activesupport/lib/active_support/core_ext/object/blank.rb).  I would go with Rubocop's recommendation here.  For " " `blank?` is true and `present?` is false.  If you want to allow filenames that include `" "` you would have to explicitly test for that.  However, that seems so edge casey that I think it's a bad idea to consider in our code.  If you know this is actually a likely name, then I'd reject it or rename it to something reasonable.